tests/Api/ApiTest.php

```diff
@@ -131,20 +234,216 @@ public function testCallLocale()
 			}
 		]);
 
+		$de = ['de', 'de_DE', 'de_DE.UTF-8', 'de_DE.UTF8', 'de_DE.ISO8859-1'];
 		$this->assertSame('something', $api->call('foo'));
-		$this->assertTrue(in_array(setlocale(LC_MONETARY, 0), ['de', 'de_DE', 'de_DE.UTF-8', 'de_DE.UTF8', 'de_DE.ISO8859-1']));
-		$this->assertTrue(in_array(setlocale(LC_NUMERIC, 0), ['de', 'de_DE', 'de_DE.UTF-8', 'de_DE.UTF8', 'de_DE.ISO8859-1']));
-		$this->assertTrue(in_array(setlocale(LC_TIME, 0), ['de', 'de_DE', 'de_DE.UTF-8', 'de_DE.UTF8', 'de_DE.ISO8859-1']));
+		$this->assertTrue(in_array(setlocale(LC_MONETARY, 0), $de));
+		$this->assertTrue(in_array(setlocale(LC_NUMERIC, 0), $de));
+		$this->assertTrue(in_array(setlocale(LC_TIME, 0), $de));
 		$this->assertSame($originalLocale, setlocale(LC_CTYPE, 0));
```

Comment:
@lukasbestle `With priority Z/not urgent`: This unit test now fails as we have added the `setUp` method setting an `app` object with a locale. At least that's why I think it fails. How would we need to alter this test method to pass but still ensure it checks the right things? Not fully understanding all the `setLocale` calls, so if you have an idea sometime, not urgent at all.